### PR TITLE
[FIX] Rename wiki_cn/wiki_multilingual to wiki_cn_cased/wiki_multilingual_uncased

### PR DESCRIPTION
**Pull request by kenjewu**
## Description ##
Rename`wiki_multilingual`/`wiki_cn` to `wiki_multilingual_uncased`/`wiki_cn_cased`.
#491 
### Essentials ###
- [x] PR's title starts with a category (e.g. [BUGFIX], [MODEL], [TUTORIAL], [FEATURE], [DOC], etc)
- [x] Changes are complete (i.e. I finished coding on this PR)
- [ ] All changes have test coverage
- [x] Code is well-documented

### CODE REVIEW

**Comment by kenjewu** [comment]
Now I have modified all the old names directly. The program will throw ` DeprecationWarning ` if `wiki_multilingual`/`wiki_cn` is used by the user. Do we need to temporarily keep the old name available, just give a warning? 
In addition, the data server needs to update the name of the file for downloading.

File: src/gluonnlp/model/bert.py [hunk 1]
```diff
@@ -638,6 +639,10 @@ def get_bert_model(model_name=None, dataset_name=None, vocab=None,
                           use_residual=predefined_args['use_residual'])
     # bert_vocab
     from ..vocab import BERTVocab
+    if dataset_name in ['wiki_cn', 'wiki_multilingual']:
```
Comment (eric-haibin-lin) [inline src/gluonnlp/model/bert.py]:
```
warnings.warn('wiki_cn/wiki_multilingual will be deprecated. Please use wiki_cn_cased/wiki_multilingual_uncased instead').
```

File: src/gluonnlp/model/bert.py [hunk 2]
```diff
@@ -460,9 +460,9 @@ def _decode(self, sequence, masked_positions):
     {name: checksum for checksum, name in [
         ('5656dac6965b5054147b0375337d5a6a7a2ff832', 'bert_12_768_12_book_corpus_wiki_en_cased'),
         ('75cc780f085e8007b3bf6769c6348bb1ff9a3074', 'bert_12_768_12_book_corpus_wiki_en_uncased'),
-        ('237f39851b24f0b56d70aa20efd50095e3926e26', 'bert_12_768_12_wiki_multilingual'),
```
Comment (eric-haibin-lin) [inline src/gluonnlp/model/bert.py]: Keep the original one and add a new entry with cased/uncased as the suffix

File: src/gluonnlp/data/utils.py [hunk 3]
```diff
@@ -221,8 +223,8 @@ def _slice_pad_length(num_items, length, overlap=0):
                'book_corpus_wiki_en_cased': '2d62af22535ed51f35cc8e2abb607723c89c2636',
                'book_corpus_wiki_en_uncased': 'a66073971aa0b1a262453fe51342e57166a8abcf',
                'wiki_multilingual_cased': '71bb9e248dc75dce9227d3c8c16fde3993588b9e',
-               'wiki_cn': 'a1e06f8e39ae51ab8a92b8458e6a658b8b1f72bf',
```
Comment (eric-haibin-lin) [inline src/gluonnlp/data/utils.py]: Keep the original one and add a new entry with cased/uncased as the suffix

File: scripts/bert/finetune_classifier.py [hunk 4]
```diff
@@ -344,9 +344,9 @@ def train(metric):
                     metric_val = [metric_val]
                 eval_str = '[Epoch {} Batch {}/{}] loss={:.4f}, lr={:.7f}, metrics=' + \
                     ','.join([i + ':{:.4f}' for i in metric_nm])
-                logging.info(eval_str.format(epoch_id + 1, batch_id + 1, len(train_data), \
-                    step_loss / args.log_interval, \
-                    trainer.learning_rate, *metric_val))
+                logging.info(eval_str.format(epoch_id + 1, batch_id + 1, len(train_data),
+                                             step_loss / args.log_interval,
+                                             trainer.learning_rate, *metric_val))
```
Comment (szha) [inline scripts/bert/finetune_classifier.py]: nit: logging allows lazy construction of the string. you can make use of that by doing
`logging.info(eval_str, epoch_id + 1, batch_id + 1, ...)`. As a result, the logging string is not actually constructed unless logging actually happens, which saves the construction of string if the logger is configured to do nothing.

**Comment by mli** [comment]
Job PR-594/9 is complete. 
Docs are uploaded to http://gluon-nlp-staging.s3-accelerate.dualstack.amazonaws.com/PR-594/9/index.html

**Pull request merged**